tests/unit/header/Header.spec.js

```diff
@@ -0,0 +1,13 @@
+import { shallowMount } from '@vue/test-utils'
+import Header from '@/components/header/Header.vue'
+
+describe('HelloWorld.vue', () => {
+  const wrapper = shallowMount(Header, {
+    mocks: {
+      $t: msg => msg
+    }
```

Comment:
It is not necessary to mock the translation function as we are not using internationalization